src/components/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import './Footer.css'
+import facebookImg from '../assets/facebook.svg';
+import twitterkImg from '../assets/twitter.svg';
+import youtubeImg from '../assets/youtube.svg';
+import instagramImg from '../assets/instagram.svg';
+
+const Footer = ({className=''}) => {
+  className = className;
+
+  return (
+  <div className={className}>
```

Comment:
footer 태그를 사용해주는 게 좋겠네요~
```suggestion
  <footer className={className}>
```